### ISSUE TITLE
Poor contrast on submission interface

### ISSUE THREAD

**Issue by williamjallen** [Submitty/Submitty#8250]
**Describe the bug**
When submissions are not allowed, a dark gray banner with black text is displayed on the submission interface.  This banner has poor contrast and does not match the surrounding interface in light mode.

![image](https://user-images.githubusercontent.com/16820599/182425179-3f2201ad-303e-4da1-9a14-09ae51b3ea24.png)

**Expected behavior**
The contrast should be improved by changing the color of the background for this banner.  Personally, I think the usual Submitty blue we use for other banners would be acceptable, but other colors might be viable as well.

**Comment by bmcutler** [comment]
I don't think it should be the normal Submitty information/action blue.
It also shouldn't be red (or orange), which would suggest the user did something wrong.
I think dark grey is a good noticeable neutral 'no action possible' color.  But yes, to fix the contrast the text should be white.  Dark mode users can argue about what color scheme is appropriate there.

**Comment by rohnsha0** [comment]
@bmcutler Agreed!

**Comment by Gage-Zahn** [comment]
I am Interested in taking on this issue. I am new to this project, but I have looked for the matching style. I plan to do as follows:

Background-color: HEX #0a6496
Text Color: White  

This seems consistent with the rest of the application.
